pkg/sql/schemachanger/scbuild/testdata/create_index

```diff
@@ -34,7 +34,7 @@ CREATE INVERTED INDEX CONCURRENTLY id2
     indexId: 2
     name: id2
     tableId: 104
-- [[SecondaryIndex:{DescID: 104, IndexID: 2}, PUBLIC], ABSENT]
+- [[SecondaryIndex:{DescID: 104, IndexID: 2, ConstraintID: 0}, PUBLIC], ABSENT]
```

Comment:
In addition to the comments that Andrew already made, perhaps it's worth modifying `screl.ElementString` to not display zero-valued attrs in the element's string representation. We hadn't done this yet simply because hasn't been an issue until now.